src/execution_plan/ops/op_unwind.h

```diff
@@ -15,6 +15,11 @@
 typedef struct {
 	OpBase op;
 	SIValue list;         // List which the unwind operation is performed on.
+	SIValue from;         // From range
+	SIValue to;           // To range
+	SIValue step;         // Step range
+	SIValue current;      // current range
+	bool is_range;
 	AR_ExpNode *exp;      // Arithmetic expression (evaluated as an SIArray).
 	uint listIdx;         // Current list index.
 	int unwindRecIdx;     // Update record at this index.
```

Comment:
As a general question, I'm interested in understanding the performance implications of this change,
I'll assume we got a nice speed up for large ranges in addition to memory reduction